grammars/php.cson

```diff
@@ -331,10 +331,16 @@
           '7':
             'name': 'punctuation.section.array.begin.php'
           '8':
-            'name': 'punctuation.section.array.end.php'
+            'patterns': [
+              {
+                'include': '#parameter-default-types'
+              }
+            ]
           '9':
+            'name': 'punctuation.section.array.end.php'
+          '10':
             'name': 'invalid.illegal.non-null-typehinted.php'
-        'match': '(?xi)\n\t\t\t\t\t\t\t\\s*(array|callable) # Typehint\n\t\t\t\t\t\t\t\\s*(&)? \t\t\t\t\t# Reference\n\t\t\t\t\t\t\t\\s*((\\$+)[a-z_\\x{7f}-\\x{ff}][a-z0-9_\\x{7f}-\\x{ff}]*) # The variable name\n\t\t\t\t\t\t\t(?:\n\t\t\t\t\t\t\t\t\\s*(?:(=)\\s*(?:(null)|(\\[)\\s*(\\])|((?:\\S*?\\(\\))|(?:\\S*?))))\t# A default value\n\t\t\t\t\t\t\t)?\n\t\t\t\t\t\t\t\\s*(?=,|\\)|/[/*]|\\#|$) # A closing parentheses (end of argument list) or a comma or a comment\n\t\t\t\t\t\t\t'
+        'match': '(?xi)\n\t\t\t\t\t\t\t\\s*(array|callable) # Typehint\n\t\t\t\t\t\t\t\\s*(&)? \t\t\t\t\t# Reference\n\t\t\t\t\t\t\t\\s*((\\$+)[a-z_\\x{7f}-\\x{ff}][a-z0-9_\\x{7f}-\\x{ff}]*) # The variable name\n\t\t\t\t\t\t\t(?:\n\t\t\t\t\t\t\t\t\\s*(?:(=)\\s*(?:(null)|(\\[)((?>[^\\[\\]]+|\\[\\g<8>\\])*)(\\])|((?:\\S*?\\(\\))|(?:\\S*?))))\t# A default value\n\t\t\t\t\t\t\t)?\n\t\t\t\t\t\t\t\\s*(?=,|\\)|\/[\/*]|\\#|$) # A closing parentheses (end of argument list) or a comma or a comment\n\t\t\t\t\t\t\t'
```

Comment:
And few more questions, sorry i'm novice to git. Should i'm make this as new pull request? Or there some way to alter/replace previous one?

Also for "\n\t\t\t\t\t\t\t" occurrences, i'm basically can clean whole file, it not a big problem, but i'm not know how to merge it with fixes commits, since there is 3 versions now. Should i'm update it in branches and in original/master as well or there more simple way to change in branches(fixes one) only and somehow transfer to master?

Some docs on how to for this questions will be really nice.